target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -239,3 +239,13 @@ define Device/tplink_tl-wr2543-v1
   SUPPORTED_DEVICES += tl-wr2543-v1
 endef
 TARGET_DEVICES += tplink_tl-wr2543-v1
+
+define Device/tplink_cpe210-v3
+  $(Device/tplink-safeloader)
+  ATH_SOC := qca9533
+  DEVICE_TITLE := TP-Link CPE210 v3
+  DEVICE_PACKAGES := rssileds
+  TPLINK_BOARD_ID := CPE210V3
+  IMAGES := sysupgrade.bin factory.bin
```

Comment:
This IMAGES line can be dropped. Device/tplink-safeloader includes Device/tplink, which has added factory.bin to IMAGES

Reply:
done